sirepo/template/radia.py

```diff
@@ -43,11 +92,11 @@
 _SIM_DATA, SIM_TYPE, _SCHEMA = sirepo.sim_data.template_globals()
 _SDDS_INDEX = 0
 
+#DATA_REQUESTER = DataRequester()
+EXPORTER = Exporter()
```

Comment:
just use a class method. python is not like java, class methods work well.

Reply:
608f421